src/SFA.DAS.Reservations.Functions.RefreshCourse/Startup.cs

```diff
@@ -95,8 +98,9 @@ public IServiceProvider Build()
 
             services.AddTransient<IGetCoursesHandler, GetCoursesHandler>();
             services.AddTransient<IStoreCourseHandler, StoreCourseHandler>();
-
-            services.AddDbContext<ReservationsDataContext>(options => options.UseSqlServer(config.ConnectionString));
+            
+            services.AddDbContext<ReservationsDataContext>(options => DatabaseInjectionHelper.GetDataContext(config.ConnectionString, Configuration["EnvironmentName"]));
```

Comment:
could we do this in an extension method like here: 
https://github.com/SkillsFundingAgency/das-content-api/blob/8abd8906a254f26b52ff3fc839eb63f9dbecd570/src/SFA.DAS.ContentApi.Api/Extensions/AddDatabaseExtension.cs

no need for the InjectionHelper then

Reply:
No we have 3 function Apps having them share same Service collection extension leads to sql connection been closed randomly